src/python/pants/backend/jvm/subsystems/rsc_native_image.py

```diff
@@ -0,0 +1,31 @@
+# coding=utf-8
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from pants.backend.jvm.subsystems.zinc import _ZINC_COMPILER_VERSION
+from pants.binaries.binary_tool import NativeTool
+from pants.util.memo import memoized_method
+
+
+class RscNativeImage(NativeTool):
+  options_scope = 'rsc-native-image'
+
+  default_version = _ZINC_COMPILER_VERSION
+  name = 'rsc-pants-native-image'
+
+  @classmethod
+  def register_options(cls, register):
+    super(RscNativeImage, cls).register_options(register)
+
+    register('--native-image', fingerprint=True, type=bool,
```

Comment:
This option name could be better.

Reply:
See above: if this subsystem were in the `rsc` scope, the option name would be more useful.